frontend/src/components/TodoTaskList/types.ts

```diff
@@ -0,0 +1,11 @@
+export type TodoTaskProps = {
```

Comment:
Jeśli tutaj chcesz eksportować propsy/typy, to warto użyć dokładniejszej nazwy, która pomoże nam w refactoringu.
Prawdopodobnie będziesz chciał tutaj końcowo przechowywać tylko propsy dotyczące `TodoTaskList`, a propsy dotyczące poszczególnych itemów będziemy przechowywać już w komponencie ToDoTaska (componentu tego itemu) :)
W takiej sytuacji może dobrze byłoby nazwać plik po prostu "TodoTaskProps.ts"?

Reply:
Done. Ja w pracy mam teraz taką konwencję, że tylko folder jest z nazwy komponentem a w środku każdego pliki: hook.ts (o ile jest jakiś), index.tsx, types.ts i pliki ze stylami.